### DISCUSSION TITLE
The Hai diplomat and her clone

### DISCUSSION THREAD

**Discussion by tuuktuuk** [endless-sky/endless-sky#10781] [Lore]
I was carrying the Hai diplomat, called Saryo or somesuch, aboard my dreadnought, Surumetsa, when I joined the meeting discussing the Hai's discovery in human mainstream media. And there was a diplomat woman, with the name Saryo (or somesuch) sitting in the room and she was introduced to me as if she was a new character. Does she have a twin? Or do all Hai diplomats have the same name? If yes, why?

**Comment by ziproot** [comment]
This is a known issue.
#8820
#8770
